### DISCUSSION TITLE
Performance ModelSerializer vs DataclassSerializer

### DISCUSSION THREAD

**Discussion by Kub-AT** [oxan/djangorestframework-dataclasses#49] [General]
Can someone provide any performance benchmark of ModelSerializer vs DataclassSerializer ?
take a look: https://hakibenita.com/django-rest-framework-slow

Fyi. [@hakib](https://github.com/hakib)

**Comment by oxan** [comment]
See my comment [here](https://github.com/oxan/djangorestframework-dataclasses/issues/37#issuecomment-754631039), I don't expect performance to be great.

(I converted your issue to a discussion, as issues are only for concrete actionable items).

**Comment by henrikzabel** [comment]
Any changes to this? Have you made any benchmarks or test? 🧐
